lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -437,16 +454,20 @@ bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
                 addedDagPaths->push_back(MDagPath::getAPathTo(obj));
                 currentAddedDagPaths.append(MDagPath::getAPathTo(obj));
                 fromSdfPaths.push_back(pathsIter->GetPrimPath());
+                sdfToDagMap[pathsIter->GetPrimPath()] = MDagPath::getAPathTo(obj);
```

Comment:
Why do you need to do this here? Wasn't everything already collected in the loop above?